fix host gcc warnings

Remove gcc warnings when gcc is v3.2

Upstream-Status: Pending

Signed-of-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.ac b/configure.ac
index f5a5b74..4a20154 100644
--- a/configure.ac
+++ b/configure.ac
@@ -605,11 +605,6 @@ AC_ARG_ENABLE([werror],
     fi],
     [])
 
-if test -n "`gcc --version | grep ' 3.2'`" ; then
-        CC_WARNINGS=`echo $CC_WARNINGS | sed -e 's/-Wno-strict-aliasing//g'`
-        CXX_WARNINGS=`echo $CC_WARNINGS | sed -e 's/-Wno-strict-aliasing//g'`
-fi
-
 case $host_os in
     solaris*)
         CC_WARNINGS=`echo $CC_WARNINGS | sed -e 's/-Wcast-qual//g'`
